instrumentation/jms/src/test/java/brave/jms/ITJms_1_1_TracingMessageProducer.java

```diff
@@ -191,7 +191,7 @@ void should_record_properties(Map<String, String> producerTags) throws Exception
 
   @Test public void should_record_error() throws Exception {
     tracedSession.close();
-    should_record_error(() -> messageProducer.send(message));
+    should_record_error(() -> messageProducer.send(jms.destination, message));
```

Comment:
@anuraaga for example, the error assertion showed that the earlier tests were bad syntax. the error returned had nothing to do with the session that was caused. it is really important to not undo this as there are still a boatload of hidden bugs to find.